csharp/src/Google.Protobuf.Conformance/Google.Protobuf.Conformance.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
```

Comment:
other projects target net451 and netcoreapp1.0, so not sure if this was intended (it's probably just fun, only wanted to point that out).